android/java/org/chromium/chrome/browser/crypto_wallet/adapters/WalletCoinAdapter.java

```diff
@@ -198,7 +204,26 @@ public void onCheckedChanged(
                 if (walletListItemModel.hasNftImageLink()
                         && ImageLoader.isSupported(nftDataModel.nftMetadata.mImageUrl)) {
                     String url = nftDataModel.nftMetadata.mImageUrl;
-                    ImageLoader.createLoadNftRequest(url, context, false).into(holder.iconImg);
+                    ImageLoader.createLoadNftRequest(url, context, false)
```

Comment:
This is not doing anything specific to NFT and should be reused so rename this to something like [loadImg](https://github.com/brave/brave-core/blame/428e0b7f0db179ee538a30573c55e48aad325b2d/android/java/org/chromium/chrome/browser/crypto_wallet/util/ImageLoader.java#L32) or `createRequest`

Reply:
Let's rename it to `createRequest` then.